Dockerfile

```
@@ -1,18 +1,40 @@
 ARG RUN
 
-FROM node:lts as builderenv
+FROM amd64/ubuntu as builderlibrary
+
+WORKDIR /app
+
+RUN apt-get update
+RUN apt-get -y -qq install make build-essential git
+
+RUN git clone https://github.com/BinomialLLC/crunch.git
+RUN ls && cd crunch/crnlib && make
+
+FROM amd64/ubuntu as builderenv
 
 WORKDIR /app
 
 # some packages require a build step
 RUN apt-get update
-RUN apt-get -y -qq install python-setuptools python-dev build-essential
+RUN apt-get -y -qq install python-setuptools build-essential
 
 # We use Tini to handle signals and PID1 (https://github.com/krallin/tini, read why here https://github.com/krallin/tini/issues/8)
 ENV TINI_VERSION v0.19.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
 RUN chmod +x /tini
 
+# install yarn
+RUN apt-get update
+RUN apt-get install -y gnupg curl
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update
+RUN apt-get install -y yarn
+
+# update nodejs
+RUN curl -sL https://deb.nodesource.com/setup_19.x | bash -
```

```suggestion
RUN curl -sL https://deb.nodesource.com/setup_19.x | bash -
```
Any reason to use 19 instead of 18?

---

I forgot updating this, I was trying to replicate my exact local environment to address an issue with the building. Pushing a commit that will change this to v18 soon.